### ISSUE TITLE
Ajouter Liste de sigles de l'Organisation des Nations unies de Wikipedia

### ISSUE THREAD

**Issue by ColinMaudry** [ColinMaudry/dictionnaire-sigles#21]
https://fr.wikipedia.org/wiki/Liste_de_sigles_de_l%27Organisation_des_Nations_unies

**Comment by pachevalier** [comment]
Bof. A mon avis, il faut restreindre le champ à l'administration française. Sinon, c'est infini.

**Comment by ColinMaudry** [comment]
Oui, mais ce sont des sigles très présents. J'ai par exemple ajouté THD (Trés Haut Débit) car c'est un sigle utilisé au coeur de certaines politiques publiques.

Après, c'est surtout la question de savoir quel temps je veux y passer :)
